roles/common/templates/default.j2

```diff
@@ -5,3 +5,8 @@ DATABASE_PASSWORD={{ database_password }}
 SECRET_TOKEN={{ secret_token }}
 MAIL_LINK_HOST=www.timeoverflow.org
 MAILER_SENDER={{ mailer_sender }}
+SMTP_ADDRESS=smtp.mailgun.org
+SMTP_PORT=587
+SMTP_DOMAIN=staging.timeoverflow.org
```

Comment:
Use variable

Reply:
By now, we only use a var for the SMTP_DOMAIN, the other vars are the same in the different environments.